### PR TITLE
Added env var. Remove unused import + cargo fmt.

### PR DESCRIPTION
**Pull request by wmorgue**
👋🏻
Also added a fast build manual because project has many dependencies and `cargo update`.

### CODE REVIEW

File: .cargo/config_fast_build [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Rename this file to `config.toml` to enable "fast build" configuration. Please read the notes below.
```
Comment (dashed) [inline .cargo/config_fast_build:1]: @wmorgue where did this file come from?

Comment (wmorgue) [reply .cargo/config_fast_build:1]: I added this file for the manual on how to build the project quickly due to the large number of dependencies. [Source URL](https://github.com/bevyengine/bevy/blob/master/.cargo/config_fast_builds).
You can learn about cargo futures from the book - https://doc.rust-lang.org/cargo/reference/config.html

Comment (wmorgue) [reply .cargo/config_fast_build:1]: Any news?
cc @dashed

Comment (dashed) [reply .cargo/config_fast_build:1]: @wmorgue I think it's fine.

@thinkocapo @adam000034 @sentryadam0000345 I'm unable to merge this as I don't have permissions to this org; would this be good to merge?

Comment (thinkocapo) [reply .cargo/config_fast_build:1]: @wmorgue this is an app used internally by a team, any changes need to be reviewed by the team. Currently, the demo meets our needs.

We recommend you follow https://docs.sentry.io/platforms/rust/ for instructions on how to use the Sentry Rust SDK, as your app and environment may differ from what we're doing in sentry-demos. Appreciate your idea for reducing the dependencies.

What is this change for?
```
# NOTE: For maximum performance, build using a nightly compiler
# If you are using rust stable, remove the "-Zshare-generics=y" below.

[target.x86_64-unknown-linux-gnu]
linker = "/usr/bin/clang"
rustflags = ["-Clink-arg=-fuse-ld=lld", "-Zshare-generics=y"]

# NOTE: you must manually install https://github.com/michaeleisel/zld on mac. you can easily do this with the "brew" package manager:
# `brew install michaeleisel/zld/zld`
[target.x86_64-apple-darwin]
rustflags = ["-C", "link-arg=-fuse-ld=/usr/local/bin/zld", "-Zshare-generics=y", "-Zrun-dsymutil=no"]

[target.x86_64-pc-windows-msvc]
linker = "rust-lld.exe"
rustflags = ["-Zshare-generics=y"]

# Optional: Uncommenting the following improves compile times, but reduces the amount of debug info to 'line number tables only'
# In most cases the gains are negligible, but if you are on macos and have slow compile times you should see significant gains.
#[profile.dev]
#debug = 1
```

Comment (wmorgue) [reply .cargo/config_fast_build:1]: This change is optional and is intended to build the project faster when there are a large number of dependencies.

**Pull request closed without merging**